### PR TITLE
Stop adding a new image in infowindow-with-image-header template [do not merge]

### PR DESCRIPTION
**Pull request by xavijam**
We have found several problems here:

- We shouldn't get first field value as url. Value should be the one that appears as src in the image tag.
- We have to stop adding magically a new  element within the infowindow element, it causes problems with custom templates.

cc @javisantana @javierarce

### CODE REVIEW

**Comment by javisantana** [comment]
is this change backwards compatible?

**Comment by xavijam** [comment]
We have to test it in staging as we talked yesterday

**Comment by javisantana** [comment]
backwards compatible with the cartodb.js API i meant

File: test/spec/geo/infowindow.spec.js [hunk 1]
```diff
@@ -425,23 +425,18 @@ describe("cdb.geo.ui.infowindow", function() {
...
-      model.set("content", { fields: fieldsWithoutURL });
-      model.set('template', '<div class="cartodb-popup header" data-cover="true"><div class="cover"></div></div>');
-      expect(view.$el.find("img").length).toEqual(0);
+    it("should not edit <img> if it is not valid", function() {
+      model.set('template', '<div class="cartodb-popup header" data-cover="true"><div class="cover"><img src="hello"/></div></div>');
+      expect(view.$(".cover img").attr('style')).toBe(undefined);
     });
 
-    it("if the theme doesn't have cover don't append the image", function() {
+    it("if the theme doesn't have cover don't edit the image", function() {
```
Comment (javierarce) [inline test/spec/geo/infowindow.spec.js]: What does "don't edit" the image mean in this spec? I'm unsure of what this spec is trying to test.

Comment (xavijam) [inline test/spec/geo/infowindow.spec.js]: I can't change the name of the function, due to backward compatibility, but we don't add any  element anymore, and in fact, the function only loads and "positionates" correctly the image, that means it edits the style of the image if it is present, of course.

**Comment by xavijam** [comment]
@javisantana, I have to make more tests, but yes, it does not change or remove any method, so "API" is not affected. It only changes the behaviour of the ```_loadCover```. In any case, as I said, I'm still testing.

**Comment by javisantana** [comment]
I'm affraid of people with custom templates (not custom from the editor, i mean real handmade)

File: test/spec/geo/infowindow.spec.js (hunk 1 above)
Comment (javierarce) [inline test/spec/geo/infowindow.spec.js]: Ah, ok. So why don't we say instead: "if the theme doesn't have a cover don't style the image"?

Comment (xavijam) [inline test/spec/geo/infowindow.spec.js]: :+1:

**Pull request closed without merging**